app/js/components/homePage/manageApps/AddAddon.jsx

```diff
@@ -0,0 +1,46 @@
+/* * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function AddAddon({ handleClear }) {
+  return (
+    <div className="container-fluid">
+      <fieldset className="scheduler-border">
+        <legend className="scheduler-border">Upload Addon Package:</legend>
+        <div className="col-sm-9">
+          <input 
+            type="file" 
+            accept=".zip,.omod"
```

Comment:
Are we ready to handle .omod files with the app?

Reply:
Not yet.